compiler/rustc_typeck/src/check/mod.rs

```diff
@@ -52,13 +52,13 @@ The types of top-level items, which never contain unbound type
 variables, are stored directly into the `tcx` typeck_results.
 
 N.B., a type variable is not the same thing as a type parameter.  A
-type variable is rather an "instance" of a type parameter: that is,
-given a generic function `fn foo<T>(t: T)`: while checking the
+type variable is an "instance" of a type parameter. That is,
```

Comment:
Does `"instance"` need to be in quotes?

Reply:
I don't think so, I interpret this as additional emphasis, so I don't particularly mind if it remains in quotes or not.